test/Directory.Build.props

```diff
@@ -3,5 +3,6 @@
   <Import Project="..\Directory.Build.props" />
   <PropertyGroup>
     <ExcludeFromSourceBuild>true</ExcludeFromSourceBuild>
+    <GenerateDocumentationFile>false</GenerateDocumentationFile>
```

Comment:
this fails like in https://github.com/dotnet/templating/pull/8067, we should probably remove it